src/com/intellij/idea/plugin/hybris/project/descriptors/RegularHybrisModuleDescriptor.java

```diff
@@ -252,29 +231,38 @@ public List<JavaLibraryDescriptor> getLibraryDescriptors() {
                 });
         }
 
+        final Project currentProject = getRootProjectDescriptor().getProject();
+        if (currentProject != null) {
+            addLibsForBackofficeModule(libs, currentProject);
+        }
+
+        if (this.isAddOn()) {
+            this.processAddOnBackwardDependencies(libs);
+        }
+
+        return Collections.unmodifiableList(libs);
+    }
+
+    private void addLibsForBackofficeModule(final List<JavaLibraryDescriptor> libs, final Project currentProject) {
         if (this.hasBackofficeModule()) {
 
+            final HybrisProjectSettingsComponent instance = HybrisProjectSettingsComponent.getInstance(currentProject);
+
             libs.add(new DefaultJavaLibraryDescriptor(
                 new File(
                     this.getRootProjectDescriptor().getHybrisDistributionDirectory(),
-                    CommonIdeaService.Companion.getInstance().getBackofficeWebInfLib(getRootProjectDescriptor().getProject())
+                    instance.getBackofficeWebInfLib()
```

Comment:
do not use static access to external fields with field names which may cause collisions.

`instance` -> `MyService.instance`